Microsoft.AVS.NFS/Microsoft.AVS.NFS.psm1

```diff
@@ -0,0 +1,161 @@
+using module Microsoft.AVS.Management
+<#
+    .SYNOPSIS
+     This function mounts the NFS datastore on all hosts in the cluster.
+
+    .PARAMETER ClusterName
+     Cluster name
+
+    .PARAMETER DatastoreName
+     Name of the NFS Datastore to be mounted on hosts in the cluster.
+    
+    .PARAMETER NfsSharePath
+     Path to NFS share.
+    
+    .PARAMETER NfsHost
+     IP of NFS server.
+
+    .EXAMPLE
+     New-NFSDatastore -ClusterName Cluster1 -DatastoreName testfileshare -NfsHost 172.24.1.8 -NfsSharePath /ankstorageacct/testfileshare/
+
+    .INPUTS
+     vCenter cluster name, Name of the NFS datastore, Path to the NFS share, IP of the NFS server.
+
+    .OUTPUTS
+     None.
+#>
+function New-NFSDatastore {
+    [CmdletBinding()]
+    [AVSAttribute(10, UpdatesSDDC = $false, AutomationOnly = $true)]

```

Comment:
AutomationOnly means it won't show up in the portal - where will this be called from? It's not wrong, I'd just like to understand the whole picture.

Reply:
We don't want customers to be aware of this command. The partners would likely either use the CLI or Azure REST API to call this command.

Reply:
Hm, this will look odd in the portal - a package with no cmdlets. Oh well.

Reply:
Is there a way we can hide the package?

Reply:
Not at present, I'll add a work item to address that.